### ISSUE TITLE
Any chance in supporting Auto Ads?

### ISSUE THREAD

**Issue by japalekhin** [mazipan/vue-google-adsense#36]
I liked how Adsense was deciding where/when to place the ads on my site. Now that I've converted to an SPA, I have no choice but to manually put ads.

**Comment by mazipan** [comment]
I don't get it what do you want.
Can you help to explain a bit more?

**Comment by japalekhin** [comment]
Have you ever heard of "Auto Ads"? So it works by inserting just one snippet of code from Adsense into your site. Nothing else. Then that script looks at your website and automatically puts ads in places it (AI) thinks will give out the best results (or so they say).

That way you don't have to manually place the "In-feed Ad", the "In-article ad", or the  "banner ad" on your site. The single script does it for you.

**Comment by Smythu** [comment]
Came here to actually ask the exact same question. Will it be supported?

**Comment by mazipan** [comment]
Can you give me the sample of code for supporting auto ads?
Or any link materials?

**Comment by mazipan** [comment]
In my opinion, if you have no dynamic placement then you can just put in your HTML file.
Under `head` tag.
And doesn't need extra library to insert this kind of ads.
But, do you have try to use the ordinary AdSense type from this library?

```js
<Adsense
    data-ad-client="ca-pub-XXXXXXXXXXXXXXXX"
    data-ad-slot="1234567890">
</Adsense>
```

I think this type doesn't require you to put in dynamic placement.

**Comment by japalekhin** [comment]
That's the thing, it doesn't work. I've tried it. I placed the auto ads code in my page and it doesn't display any ads. I tried resetting the code by removing it and adding again. No good.

I actually reverted to SSR (WordPress default) architecture to get back my ads.

Also no thanks, on manual placement. It's the old way and maybe it will on some really special really rare occasions. But once you use auto ads, you'll never look back.
